### DISCUSSION TITLE
"For updates follow @squidfunk on Twitter"

### DISCUSSION THREAD

**Discussion by kaanaksit** [squidfunk/mkdocs-material#3094] [Q&A] [answered]
Sorry if the question that I am about to ask is a duplicate of a previous question. At least, my search did not return a relevant result.

I am wondering how did you manage to add:

```
For updates follow @squidfunk on Twitter
```

at the top of your page. I would like to do something similar on my site built on mkdocs-material. Thanks a billion, times for making mkdocs-material happen, it is a life saver, super easy to use!

**Comment by squidfunk** [comment, marked as answer]
The [announcement bar](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-header/#announcement-bar) is what you're looking for.
